packages/mobile/src/harmony-native/components/Artwork.tsx

```diff
@@ -0,0 +1,147 @@
+import { useEffect, useState } from 'react'
+
+import { useTheme } from '@emotion/react'
+import FastImage from 'react-native-fast-image'
+import Animated, {
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming
+} from 'react-native-reanimated'
+
+import type { FastImageProps } from './FastImage/FastImage'
+import { Skeleton } from './Skeleton'
+import { Box } from './layout/Box/Box'
+import type { BoxProps } from './layout/Box/types'
+import { Flex } from './layout/Flex/Flex'
+
+const AnimatedFastImage = Animated.createAnimatedComponent(FastImage)
+
+export type ArtworkProps = {
+  isLoading?: boolean
+  borderWidth?: number
+  'data-testid'?: string
+  noLoading?: boolean
+} & Partial<Pick<FastImageProps, 'source' | 'priority'>> &
+  BoxProps
+
+/**
+ * The artwork component displays the track cover art and appears in several
```

Comment:
nit: is this actually specific to tracks?
nbd, but since it's in a lib, want to imply the best usage. If I'm implementing a playlist tile, should I be using this?

Reply:
great call, def not